File: dataset/avatar/Python/Code/atcoder_ABC169_C.py
def main ( ) :
    from sys import setrecursionlimit , stdin
    from os import environ
    from collections import defaultdict , deque , Counter
    from math import ceil , floor , gcd
    from itertools import accumulate , combinations , combinations_with_replacement
    setrecursionlimit ( 10 ** 6 )
    dbg = ( lambda * something : print ( * something ) ) if 'TERM_PROGRAM' in environ else lambda * x : 0
    input = lambda : stdin.readline ( ).rstrip ( )
    LMIIS = lambda : list ( map ( int , input ( ).split ( ) ) )
    II = lambda : int ( input ( ) )
    P = 10 ** 9 + 7
    INF = 10 ** 9 + 10
    sa , sb = input ( ).split ( )
    a = int ( sa )
    b = int ( float ( sb ) * 100 + 0.1 )
    print ( a * b // 100 )
main ( )

File: output/qwen_2.5_72b_instruct/avatar/algo_based_translation/python/java/atcoder_ABC169_C.java

import java.util.Scanner;
import java.util.Map;
import java.util.HashMap;
import java.util.ArrayDeque;
import java.util.Deque;
import java.util.List;
import java.util.ArrayList;
import java.util.Collections;
import java.util.stream.Collectors;
import java.util.stream.IntStream;
import java.math.BigInteger;
import java.util.function.Function;

public class atcoder_ABC169_C {
    public static void main(String[] args) {
        Scanner stdin = new Scanner(System.in);
        Map<String, String> environ = System.getenv();
        Function<Object[], Void> dbg = environ.containsKey("TERM_PROGRAM") ? (x -> {
            System.out.println(x);
            return null;
        }) : (x -> null);
        Function<Void, String> input = () -> stdin.nextLine().strip();
        Function<String, List<Integer>> LMIIS = (s) -> List.of(input.apply(null).split(" ")).stream().map(Integer::parseInt).collect(Collectors.toList());
        Function<String, Integer> II = (s) -> Integer.parseInt(input.apply(null));
        int P = 1000000007;
        int INF = 1000000010;
        String[] parts = input.apply(null).split(" ");
        String sa = parts[0];
        String sb = parts[1];
        int a = Integer.parseInt(sa);
        int b = (int) (Double.parseDouble(sb) * 100 + 0.1);
        System.out.println(a * b / 100);
    }
}

